src/components/FilterForm/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import CheckIcon from 'mdi-react/CheckIcon';
```

Comment:
Now that we will keep the filter area content, we won't need the check icon. Let's bring it back to how it was previously.